README.md

```diff
@@ -62,166 +65,9 @@ docker swarm init
 
 In case you are deploying to multiple nodes, all the nodes must be connected to the same swarm. In this case, run the command above in the machine that must be the swarm manager and run `docker swarm join` in all other nodes, as instructed by the execution of `docker swarm init` in the manager machine. Check the [Docker Swarm](https://docs.docker.com/engine/swarm/) documentation for more on how to setup your cluster.
```

Comment:
Personally, I believe we should also mention `docker swarm leave`, or add a **Tear Down** section so that users know how to stop services and leave swarm :smile:

Reply:
Done.

Reply:
Done.